dali/ft/filecopy.cpp

```diff
@@ -3259,18 +3261,20 @@ void FileSprayer::updateTargetProperties()
                 {
                     FilePartInfo & curSource = sources.item(idx);
                     RemoteFilename &remoteFile = curSource.filename;
-                    splitAndStoreFileInfo(newRecord, remoteFile, idx, false);
+                    splitAndCollectFileInfo(newRecord, remoteFile, fileNames, idx, false);
```

Comment:
It may well be worth pre-sizing fileNames either before the loop, or on the 1st filename based on it's size.
e.g.
```
if (0 == idx)
{
    // pre-size fileNames buffer with estimate of total size to avoid excessive expansion/memory copying
    fileName.ensureCapacity((fileNames.length()+1) * sources.ordinality()); // estimate based on 1st, +1 for comma.
}
```

Reply:
Added